component-samples/demo/manufacturer/manufacturer-entrypoint.sh

```diff
@@ -3,6 +3,9 @@
 # Copyright 2020 Intel Corporation
 # SPDX-License-Identifier: Apache 2.0
 
+# LOG4J2 configuration file
```

Comment:
For the containers, please move this configuration to .env files.

Reply:
Application can't pick log4j configuration from .env files, it needs to be updated in the `_JAVA_OPTIONS` variable, So sticking to the current format of specifying logger configuration file.